homeassistant/components/lutron/fan.py

```diff
@@ -0,0 +1,123 @@
+"""Support for Lutron fans."""
+import logging
+
+from homeassistant.components.fan import (SPEED_LOW,
+                                          SPEED_MEDIUM,
+                                          SPEED_HIGH,
+                                          SPEED_OFF,
+                                          SUPPORT_SET_SPEED,
+                                          FanEntity)
+from . import LUTRON_CONTROLLER, LUTRON_DEVICES, LutronDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+SPEED_MEDIUM_HIGH = 'medium_high'
+SPEED_MAPPING = {
+    SPEED_OFF: 0,
+    SPEED_LOW: 25,
+    SPEED_MEDIUM: 50,
+    SPEED_MEDIUM_HIGH: 75,
+    SPEED_HIGH: 100
+}
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Lutron fans."""
+    devs = []
```

Comment:
Please add a guard clause here that checks if discovery_info is None and return if so.